tasks/publish-pyxis-repository/publish-pyxis-repository.yaml

```diff
@@ -72,6 +78,12 @@ spec:
             exit 1
         fi
 
+        SOURCE_CONTAINER_ENABLED="false"
```

Comment:
How about renaming the var to `ENABLE_SOURCE_CONTAINER`? I think it would be clearer that it controls whether or not we run the curl request (to enable it). Because if `false`, we don't do anything.

Reply:
fixed